atest/robot/libdoc/libdoc_resource.robot

```diff
@@ -189,6 +189,12 @@ Keyword Tags Should Be
     ${tags}=   Get Elements Texts    ${kws}[${index}]    xpath=tags/tag
     Should Be Equal    ${tags}    ${expected}
 
+Resource Tags Should Be
+    [Arguments]    ${index}    @{expected}
+    ${tags}    Get Elements Texts    ${LIBDOC}    xpath=tags
+    ${tags}    Split String    ${tags}[0]    \n
+    List Should Contain Sub List    ${tags}    ${expected}
+
```

Comment:
And the atests only pass because you are using `List should contain sub list` because `?!?!??` from your test is not in `@{expected}` and therefore not checket at all. It is in `@{index}`

Reply:
Yes, You are right about `${index}`, I dropped it, because getting tags for xml once, instead of multiple times is more efficient. Then using list comparison is right choice, I believe. Here is how it works:
![image](https://user-images.githubusercontent.com/35737166/103335063-1988f580-4a74-11eb-966b-2a12dbc1608a.png)
All tags are checked, except those two `''` from list, that where created by splitting string. And split string was used because all tags where combined with `\n`.

Thanks for he suggestion for name, placement ad other files that needs to be tested.